src/main/java/codewars/tasks/java/Main.java

```diff
@@ -4,12 +4,28 @@
 public class Main {
 
     public static void main (String [] args){
+
+        /**
+         * This method counts length of a vector with given params
+         */
```

Comment:
Java doc comment should be the above method. If you what to clarify code it would better use commented line